specs/vep-1243-vdk-meta-jobs/README.md

```diff
@@ -153,6 +188,107 @@ Consider at least the below topics but you do not need to cover those that are n
   * What secrets are needed by the components? How are these secrets secured and attained?
 -->
 
+### Workflow
+
+The structure of the Meta Job is a DAG. Here is an example of the DAG of jobs workflow:
+1. The Meta Job(DAG) is initialized, following the preset configuration.
+2. The DAG is being validated that there are no conflicts regarding the existence of the team/jobs in the
+Control Service it addresses as well as any other technical errors.
+3. If there are no errors during step 2., the DAG is built.
+4. The DAG of jobs is executed, taking into account all the order information.
+5. A summary of the DAG is logged.
+
+### Configuration details
+
+You are allowed to specify whether the Meta Job itself should fail if any of its orchestrated jobs encounter errors.
+This is achieved by adding the Data Job attribute **fail_meta_job_on_error**. Set to true, as by default, would mean
+that in the event of a failing orchestrated job, the Meta Job would also fail - with USER error.
+This can help to prevent cascading failures and ensure that issues are isolated and addressed as soon as possible.
+
+There are more settings that are configurable by the user, that can be explored in the
+[config](/projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_configuration.py).
+
+### Capacity Estimation and Constraints
```

Comment:
The purpose of the section is to help people understand what are the expected limits. They would help organizations ensure that their systems are designed to handle the expected workload and meet performance expectations and predict resources. 

The main resource meta jobs take is API requests . 
So we should explain when API requests are made. 

VDK when it runs DAG 

* It makes API request to start a job 
* It makes API request to check status of a job  (based on META_JOBS_TIME_BETWEEN_STATUS_CHECK_SECONDS)
* It makes API request to get details after job is completed 
* On re-tries (if job start failed) 

Let's create a formula for calculating the number of API requests in a DAG run (assuming no retries needed). We'll use the following variables:

J: Number of jobs in the DAG
E: Average execution duration of a jobs 
S: META_JOBS_TIME_BETWEEN_STATUS_CHECK_SECONDS

Generally job1 would not impact the number of requests job2 would make for its duration. 

So 1 job would make 1 + E / S + 1 API request for its run. 

hence the dag would make J * (1 + E/S + 1) total requests. 

For for dag of 6 jobs with average duration 10 minutes and default 40 seconds between checks. 

6 * (1 + 600/40 + 1) = 102 

That is a bit simpler calculation isn't ?

Reply:
Totally! Great simplification! Will address this.